packages/manager/modules/netapp/src/dashboard/index/template.html

```diff
@@ -6,8 +6,58 @@
         >
             <oui-tile-definition
                 term="{{:: 'netapp_dashboard_information_name' | translate }}"
-                description="{{ $ctrl.storage.name }}"
             >
+                <oui-tile-description data-ng-if="!$ctrl.isEditingName">
+                    {{ $ctrl.storage.name }}
+                </oui-tile-description>
+                <oui-tile-actions data-ng-if="!$ctrl.isEditingName">
+                    <button
+                        type="button"
+                        class="oui-button oui-button_ghost"
+                        data-ng-click="$ctrl.toggleNameEdition()"
+                    >
+                        <span class="oui-icon oui-icon-pen"></span>
+                    </button>
+                </oui-tile-actions>
+                <oui-tile-description data-ng-if="$ctrl.isEditingName">
+                    <div class="oui-input-group oui-input-group_default ">
+                        <div class="oui-input-overlay">
+                            <input
+                                class="oui-input"
+                                placeholder="{{ 'netapp_dashboard_information_name' | translate }}"
+                                type="text"
+                                ng-model="$ctrl.editNameValue"
+                            />
```

Comment:
I see restriction in the translation file regarding the content of the name, but here there is no pattern, did you forgot it ?